features/app_features.feature

```diff
@@ -60,6 +60,27 @@ Scenario: A user can add initial features while editing apps
    And I follow "Edit App"
    When I fill in "App Initial Features" with "app1 features"
    And I press "Update App"
-   And I follow "app1" 
+   And I follow "app1"
    Then I should see "App Initial Features"
-   And I should see "app1 features"
\ No newline at end of file
+   And I should see "app1 features"
+
+# Testing pivotal tracker url in app create
+Scenario: A user can add pivotal tracker url while creating apps
```

Comment:
I think this is a good scenario to test and I was able to follow along with understanding what the user should experience!